entrypoint.sh

```diff
@@ -73,6 +78,8 @@ while true ; do
             echo "  --test-pkg=PKG      Run only tests from PKG"
             echo "  --pause             Sleep forever after tests execute"
             echo "  --micro-osd         Specify path to micro-osd script"
+            echo "  --wait=FILES        Wait for files before starting tests"
```

Comment:
Minor nit: perhaps naming it `--wait-for` would be clearer. Without reading the help more I would have assumed `--wait` was time based based.

Reply:
done